src/methods.js

```diff
@@ -152,6 +156,26 @@ var methods = {};
         applyToAll.call(this, rule, "activated", active);
     };
 
+    methods.ruleIsMet = function (rule) {
+        if ($.isFunction(rulesEngine.validation[rule])) {
+            if (rule === "wordLength") {
+                rule = "wordLengthStaticScore";
+            }
+
+            var rulesMetCnt = 0;
+
+            this.each(function (idx, el) {
+                var options = $(el).data("pwstrength-bootstrap");
+
+                rulesMetCnt += rulesEngine.validation[rule](options, $(el).val(), 1);
+            });
+
+            return (rulesMetCnt === this.length);
+        } else {
```

Comment:
JSLint errors:
src/methods.js:174:11	Unnecessary 'else' after disruption.

Since there is a return, the else it's not necessary, it can be removed.